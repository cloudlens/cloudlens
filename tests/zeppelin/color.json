{"paragraphs":[{"text":"%cl\nsource(\"local:tests/zeppelin/color.log\")","dateUpdated":"Jun 27, 2016 3:42:11 PM","config":{"colWidth":12,"graph":{"mode":"table","height":234,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056387725_-216677145","id":"20160627-153947_1514889862","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>message</font>:&quot;My color is blue!&quot;\t\n%html <font color=blue>message</font>:&quot;My color is red!&quot;\t\n%html <font color=blue>message</font>:&quot;My color is green!&quot;\t\n%html <font color=blue>message</font>:&quot;My color is blue!&quot;\t\n%html <font color=blue>message</font>:&quot;My color is red!&quot;\t\n%html <font color=blue>message</font>:&quot;My color is green!&quot;\t\n%html <font color=blue>message</font>:&quot;My color is red!&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>message</font>:&quot;My color is blue!&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is red!&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is green!&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is blue!&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is red!&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is green!&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is red!&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>message</font>:&quot;My color is blue!&quot;",""],["%html <font color=blue>message</font>:&quot;My color is red!&quot;",""],["%html <font color=blue>message</font>:&quot;My color is green!&quot;",""],["%html <font color=blue>message</font>:&quot;My color is blue!&quot;",""],["%html <font color=blue>message</font>:&quot;My color is red!&quot;",""],["%html <font color=blue>message</font>:&quot;My color is green!&quot;",""],["%html <font color=blue>message</font>:&quot;My color is red!&quot;",""]]},"dateCreated":"Jun 27, 2016 3:39:47 PM","dateStarted":"Jun 27, 2016 3:42:11 PM","dateFinished":"Jun 27, 2016 3:42:11 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:422"},{"text":"state = \"green\";","dateUpdated":"Jun 27, 2016 3:42:13 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056387726_-215522898","id":"20160627-153947_598298676","result":{"code":"SUCCESS","type":"TEXT","msg":"\"green\"\n"},"dateCreated":"Jun 27, 2016 3:39:47 PM","dateStarted":"Jun 27, 2016 3:42:13 PM","dateFinished":"Jun 27, 2016 3:42:13 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:423"},{"text":"%cl\nmatch{\n    \"My color is (?<color>[^ ]+)!\"\n}\n\nstream(entry){\n    if (state == \"blue\" && entry.color == \"red\") state = \"red\";\n    else if (state == \"red\" && entry.color == \"green\") state = \"green\";\n    else if (state == \"green\" && entry.color == \"blue\") state = \"blue\";\n    else state = \"error\"\n}\n\n\nstream(entry) when (entry.color){\n    if (state == \"error\")\n        print(\"Wrong pattern at color \" + entry.color + \"!\");\n}","dateUpdated":"Jun 27, 2016 3:42:32 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056387726_-215522898","id":"20160627-153947_948367452","result":{"code":"SUCCESS","type":"TEXT","msg":"Wrong pattern at color red!\n"},"dateCreated":"Jun 27, 2016 3:39:47 PM","dateStarted":"Jun 27, 2016 3:42:32 PM","dateFinished":"Jun 27, 2016 3:42:32 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:424"},{"text":"CL.log","dateUpdated":"Jun 27, 2016 3:44:29 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056387726_-215522898","id":"20160627-153947_623403957","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>message</font>:&quot;My color is blue!&quot;\t%html <font color=blue>color</font>:&quot;blue&quot;\t\n%html <font color=blue>message</font>:&quot;My color is red!&quot;\t%html <font color=blue>color</font>:&quot;red&quot;\t\n%html <font color=blue>message</font>:&quot;My color is green!&quot;\t%html <font color=blue>color</font>:&quot;green&quot;\t\n%html <font color=blue>message</font>:&quot;My color is blue!&quot;\t%html <font color=blue>color</font>:&quot;blue&quot;\t\n%html <font color=blue>message</font>:&quot;My color is red!&quot;\t%html <font color=blue>color</font>:&quot;red&quot;\t\n%html <font color=blue>message</font>:&quot;My color is green!&quot;\t%html <font color=blue>color</font>:&quot;green&quot;\t\n%html <font color=blue>message</font>:&quot;My color is red!&quot;\t%html <font color=blue>color</font>:&quot;red&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>message</font>:&quot;My color is blue!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;blue&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is red!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;red&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is green!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;green&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is blue!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;blue&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is red!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;red&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is green!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;green&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;My color is red!&quot;"},{"value":"%html <font color=blue>color</font>:&quot;red&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>message</font>:&quot;My color is blue!&quot;","%html <font color=blue>color</font>:&quot;blue&quot;",""],["%html <font color=blue>message</font>:&quot;My color is red!&quot;","%html <font color=blue>color</font>:&quot;red&quot;",""],["%html <font color=blue>message</font>:&quot;My color is green!&quot;","%html <font color=blue>color</font>:&quot;green&quot;",""],["%html <font color=blue>message</font>:&quot;My color is blue!&quot;","%html <font color=blue>color</font>:&quot;blue&quot;",""],["%html <font color=blue>message</font>:&quot;My color is red!&quot;","%html <font color=blue>color</font>:&quot;red&quot;",""],["%html <font color=blue>message</font>:&quot;My color is green!&quot;","%html <font color=blue>color</font>:&quot;green&quot;",""],["%html <font color=blue>message</font>:&quot;My color is red!&quot;","%html <font color=blue>color</font>:&quot;red&quot;",""]]},"dateCreated":"Jun 27, 2016 3:39:47 PM","dateStarted":"Jun 27, 2016 3:44:29 PM","dateFinished":"Jun 27, 2016 3:44:29 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:425"},{"dateUpdated":"Jun 27, 2016 3:39:47 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056387726_-215522898","id":"20160627-153947_611740356","dateCreated":"Jun 27, 2016 3:39:47 PM","status":"READY","progressUpdateIntervalMs":500,"$$hashKey":"object:426"}],"name":"State machines","id":"2BPBX4NJ1","angularObjects":{"2BRB4ZK9Z":[],"2BN98KZFM":[]},"config":{"looknfeel":"default"},"info":{}}
