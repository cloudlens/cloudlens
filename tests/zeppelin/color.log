My color is blue!
My color is red!
My color is green!
My color is blue!
My color is red!
My color is green!
My color is red!
