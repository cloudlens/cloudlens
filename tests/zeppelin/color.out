Wrong pattern at color red!
